pkg/storj/node.go

```diff
@@ -143,6 +144,19 @@ func (id NodeID) MarshalJSON() ([]byte, error) {
 	return []byte(`"` + id.String() + `"`), nil
 }
 
+// Value set a NodeID to a database field
+func (id NodeID) Value() (driver.Value, error) {
+	return id.Bytes(), nil
+}
+
+// Scan extracts a NodeID from a database field
+func (id *NodeID) Scan(src interface{}) (err error) {
+	b, _ := src.([]byte)
```

Comment:
check the type cast result and return an appropriate error if it's a mismatch

Reply:
done